app/src/main/java/com/kickstarter/services/KSApolloClient.kt

```diff
@@ -189,44 +189,6 @@ class KSApolloClient(val service: ApolloClient) : ApolloClientType {
         }
     }
 
-    private fun createCommentObject(commentFr: fragment.Comment?): Comment {
```

Comment:
It has been moved outside the KSApollo class as a private package method nothing more, I was perfectly able to use it for replies as well 👍